cla_public/templates/_cookie-consent-form.html

```diff
@@ -1,97 +1,97 @@
-		<h2 class="gem-c-notice__title">Your cookie settings were saved</h2>
-		<p class="govuk-body">Government services may set additional cookies and, if so, will have their own cookie policy and banner.</p>
+		<h2 class="gem-c-notice__title">{% trans %}Your cookie settings were saved{% endtrans %}</h2>
+		<p class="govuk-body">{% trans %}Government services may set additional cookies and, if so, will have their own cookie policy and banner.{% endtrans %}</p>
 		<a class="govuk_link cookie-settings__prev-page" href="#" data-module="track-click" data-track-category="cookieSettings" data-track-action="Back to previous page">
-        Go back to the page you were looking at
+        {% trans %}Go back to the page you were looking at{% endtrans %}
       </a>
 	</section>
 </div>
 
-<p class="govuk-body">Cookies are files saved on your phone, tablet or computer when you visit a website.</p>
-<p class="govuk-body">We use cookies to store information about how you use ‘Check you can get legal aid’, such as the pages you visit.</p>
+<p class="govuk-body">{% trans %}Cookies are files saved on your phone, tablet or computer when you visit a website.{% endtrans %}</p>
+<p class="govuk-body">{% trans %}We use cookies to store information about how you use ‘Check you can get legal aid’, such as the pages you visit.{% endtrans %}</p>
 
-<h2 class="govuk-heading-l">Cookie settings</h2>
+<h2 class="govuk-heading-l">{% trans %}Cookie settings{% endtrans %}</h2>
 <div class="cookie-settings__no-js">
 	<div class="gem-c-govspeak govuk-govspeak " data-module="govspeak">
-		<p class="govuk-body">We use Javascript to set most of our cookies. Unfortunately Javascript is not running on your browser, so you cannot change your settings. You can try:</p>
+		<p class="govuk-body">{% trans %}We use Javascript to set most of our cookies. Unfortunately Javascript is not running on your browser, so you cannot change your settings. You can try:{% endtrans %}</p>
 		<ul class="govuk-list">
-			<li>reloading the page</li>
-			<li>turning on Javascript in your browser</li>
+			<li>{% trans %}reloading the page{% endtrans %}</li>
+			<li>{% trans %}turning on Javascript in your browser{% endtrans %}</li>
 		</ul>
 	</div>
 </div>
 
 <div class="cookie-settings__form-wrapper">
-	<p class="govuk-body">We use different types of cookies. You can choose which cookies you're happy for us to use and how the data is shared.</p>
+	<p class="govuk-body">{% trans %}We use different types of cookies. You can choose which cookies you're happy for us to use and how the data is shared.{% endtrans %}</p>
 	<form method="get" data-module="cookie-settings">
     <div class="gem-c-govspeak govuk-govspeak " data-module="govspeak">
-      <h2 class="govuk-heading-l">Strictly necessary cookies</h2>
-      <p class="govuk-body">We use essential cookies to help you use the ‘Check you can get legal aid’ service. These do things such as:</p>
-      <ul class="govuk-list">
```

Comment:
This class should not have been removed.

Reply:
@MalcolmVonMoJ - can you suggest an alternative pls? These bullets are not currently showing on the live site: https://checklegalaid.service.gov.uk/cookie-settings

Reply:
If bullets are wanted, use `<ul class="govuk-list govuk-list--bullet">`

Reply:
This has now been done